Resources/Public/js/DragDrop.js

```diff
@@ -33,6 +39,9 @@ define(['jquery', 'jquery-ui/sortable'], function ($) {
      * initializes Drag+Drop for all content elements on the page
      */
     DragDrop.initialize = function() {
+        // setting the style of all divs with data-language-uid to 'postion:relative' in order to prevend during dragging a wrong positioning
```

Comment:
Should the last part of this comment read `in order to prevent dragging to a wrong positioning`? Looks like a word got lost or there's an extra one. And mind the typo in `prevent` :)

Overall: we don't really need the parameter annotations in JS but of course they don't hurt. We see this D&D JS patch as hopefully temporary so as long as it works, that's Good Enough (TM) for us ;)